doc/classes/Projection.xml

```diff
@@ -24,6 +24,15 @@
 			<description>
 			</description>
 		</constructor>
+		<constructor name="Projection">
+			<return type="Projection" />
+			<param index="0" name="x_axis" type="Vector4" />
+			<param index="1" name="y_axis" type="Vector4" />
+			<param index="2" name="z_axis" type="Vector4" />
+			<param index="3" name="w_axis" type="Vector4" />
+			<description>
```

Comment:
I think it might be best to document this as it is unclear how each of the "axis" arguments will impact the Projection that is formed.

Reply:
Done.